packages/flutter/lib/src/material/floating_action_button_location.dart

```diff
@@ -419,6 +419,16 @@ abstract class FloatingActionButtonLocation {
   /// ![](https://flutter.github.io/assets-for-api-docs/assets/material/floating_action_button_location_mini_end_docked.png)
   static const FloatingActionButtonLocation miniEndDocked = _MiniEndDockedFabLocation();
 
+  /// End-aligned [FloatingActionButton], floating over the
+  /// [Scaffold.bottomNavigationBar] so that the floating
+  /// action button lines up with the center of the bottom navigation bar.
+  ///
+  /// This is unlikely to be a useful location for apps that lack a [BottomNavigationBar]
+  /// or a [BottomAppBar] with olds material.
```

Comment:
"olds" => "old"

But actually, what do you mean by this?  Maybe be more explicit, like "[BottomNavigationBar], or a [BottomAppBar] before Material 3."